### DISCUSSION TITLE
Enhancement Proposal for ListView Component

### DISCUSSION THREAD

**Discussion by ghost** [adobe/react-spectrum#7501] [Q&A]
# Issue: Enhancement Proposal for ListView Component

## Description
The ListView component displays a list of interactive items, allowing users to navigate, select, or perform actions. I would like to propose some enhancements to improve the user experience.

## Proposed Enhancements

1. **Avatar or Icon Support**: 
   Instead of just using an image, it would be beneficial to provide the option to include either an Avatar component or an Icon. This would enhance the visual representation of the items in the ListView and provide a more engaging user experience.

2. **Responsive Selection Style**:
   I propose that the `selectionStyle` property, which currently accepts values of `'checkbox'` or `'highlight'`, be made responsive. The following responsive codes could be implemented:
   ```javascript
   {
       base: "highlight",
       S: "checkbox",
       M: "",
       L: "checkbox"
   }

**Comment by snowystinger** [comment]
Thanks for the feedback, I assume you're using the React Spectrum component, not RAC?

We already support Icons, see https://react-spectrum.adobe.com/react-spectrum/ListView.html#complex-items

Avatar support will be in our Spectrum 2 implementation, not sure it'll make it into the current S1 implementation.

**Reply by ghost** [reply]
Merci. I saw it is an illustration not a workflow icon. 

Good news 🗞️   spectrum 2 introduce Avatar.

Thanks 🙏👍 

the responsive highlight or check box functionality would be really great because really on a smartphone it is much easier to toggle but sometimes on a computer it is also interesting and vice versa.

**Reply by snowystinger** [reply]
> the responsive highlight or check box functionality would be really great because really on a smartphone it is much easier to toggle but sometimes on a computer it is also interesting and vice versa.

Apologies, I don't follow, what is the ask here?
